app/assets/tailwind/application.css

```diff
@@ -687,3 +687,79 @@ a.pill-focus-reset:focus-visible {
 [data-controller="pathogen--tabs"] [role="tab"][aria-selected="false"] {
   @apply border-transparent text-slate-700 hover:border-slate-700 hover:text-slate-900 dark:text-slate-200 dark:hover:border-white dark:hover:text-white;
 }
+
+/* SelectWithAutoCompleteComponent */
+
+.combobox {
```

Comment:
A better class name might be `.autocomplete-select`, `combobox` is a little too generic here.